lib/rouge/lexers/varnish.rb

```diff
@@ -4,23 +4,30 @@
 module Rouge
   module Lexers
     class Varnish < RegexLexer
-      title 'Varnish'
-      desc 'The Varnish (high-performance web accelerator) configuration language'
+      title 'VCL: Varnish Configuration Language'
+      desc 'The configuration language for Varnish HTTP Cache (varnish-cache.org)'
 
-      tag 'varnish'
-      aliases 'varnishconf', 'VCL'
+      tag 'vcl'
+      aliases 'varnishconf', 'varnish'
       filenames '*.vcl'
-      mimetypes 'text/x-varnish'
+      mimetypes 'text/x-varnish', 'text/x-vcl'
+
+      INUM = %r/[0-9]+/
+      FNUM = %r/#{INUM}+(?:\.#{INUM}(?:e[+-]?#{INUM})?|e[+-]?#{INUM})/i
```

Comment:
I think maybe my brain has just stopped working but I don't see what the union in this regex covers. Could you explain it to me?